src/renderer/components/kube-object-list-layout/kube-object-list-layout.tsx

```diff
@@ -91,6 +101,14 @@ export class KubeObjectListLayout<K extends KubeObject> extends React.Component<
               ...searchProps,
               placeholder: `Search ${placeholderString}...`,
             },
+            info: (
+              <>
+                {info}
+                {this.loadErrors.length > 0 && (
+                  <Icon material="warning" className="load-error" tooltip={this.loadErrors.join("\n")}/>
```

Comment:
it looks like the newline is not respected when the tooltip is displayed

![Screen Shot 2021-11-10 at 8 05 03 PM](https://user-images.githubusercontent.com/40840436/141218433-7a9d13cc-48ab-4b47-843c-d7b11468cd61.png)